master/static/migrations/20220512160917_update-job-type-jobs.tx.up.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TYPE public.job_type RENAME TO _job_type; 
```

Comment:
@stoksc there are new migrations in determined dated after this. Do I need to update the dates for these?